src/platform/forms-system/test/js/components/ContactInfoReview.unit.spec.jsx

```diff
@@ -47,6 +47,8 @@ describe('<ContactInfoReview>', () => {
     expect($('button.edit-page', container)).to.exist;
     expect($('h4', container).textContent).to.eq(content.title);
     expect($$('dt', container).length).to.eq(9);
+    expect($$('dd', container).length).to.eq(9);
+    expect($$('dd.dd-privacy-hidden', container).length).to.eq(9);
```

Comment:
Is this one with `dd` different than the `dd.dd-privacy-hidden`?

Reply:
I just wanted to make sure that all the `dd` include the hidden class. I mean, we could combine these 2 checks.

Reply:
I removed the plain `dd` check